views/administration/school.hbs

```diff
@@ -196,11 +196,31 @@
 								{{$t "administration.school.label.schoolNumber" }}
 							</label>
 
-							<input id="officialSchoolNumber" value="{{../school.officialSchoolNumber}}" type="text" pattern="\D{0,2}-*\d{5}$" class="form-control" name="officialSchoolNumber"
+							<input id="officialSchoolNumber" {{#if ../school.officialSchoolNumber}}disabled{{/if}} value="{{../school.officialSchoolNumber}}" type="text" pattern="\D{0,2}-*\d{5}$" class="form-control" name="officialSchoolNumber"
 								placeholder="12345" {{#if ../schoolUsesLdap}} readonly{{/if}}
 								{{#if ../isExpertSchool}} readonly{{/if}} {{#userHasPermission 'SCHOOL_EDIT'}}{{else}}
 								readonly{{/userHasPermission}}/>
 						</div>
+						<div class="form-group">
+						<label>{{$t "administration.school.label.chooseACounty"}}</label>
+						<select class="form-control" name="county"  placeholder="Kreis"
```

Comment:
right, the select should have an id attribute, so then it can be referenced by the label